sources/engine/Xenko.Graphics/FastTextRenderer.cs

```diff
@@ -191,7 +190,8 @@ public void End([NotNull] GraphicsContext graphicsContext)
             // Set the rendering parameters
             simpleEffect.Parameters.Set(TexturingKeys.Texture0, DebugSpriteFont);
             simpleEffect.Parameters.Set(SpriteEffectKeys.Color, TextColor);
-
+            simpleEffect.Parameters.Set(SpriteBaseKeys.MatrixTransform, MatrixTransform);            
```

Comment:
Remove trailing spaces (just trying out the _suggestion_ feature to be honest :smile: )
```suggestion
            simpleEffect.Parameters.Set(SpriteBaseKeys.MatrixTransform, MatrixTransform);
```